resources/assets/js/pages/home.js

```diff
@@ -19,6 +19,11 @@ window.coreApiLoaded = (api) => {
  * @return void.
  */
 window.initMap = () => {
+    function verifyPageLoadedProperly(){
```

Comment:
There was a long standing bug where sometimes the JS would not load properly on the site and that caused it to crash. This is a workaround of that issue

Reply:
How does this affect the function later in the file (https://github.com/ritstudentgovernment/laravel-naps/blob/a0ee02460089ff7eb6e4016ca3a4937b835e4ac5/resources/assets/js/pages/home.js#L79) that attempts to reload the map if it hangs?

Also the function is set to reload if the map is still loading after 2 seconds. If a user has a slow connection, does this result in the map loading forever?

Reply:
Im not sure if we need that still, with the solution that I implemented here. Thanks for catching that, ill test it and see if I get any issues.

Reply:
I have not run into the issue with that secondary check removed. Good catch, ill push up the change